extensions/wikia/BannerNotifications/BannerNotifications.setup.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Docs are on internal in UI Style Guide
+ */
+$wgExtensionCredits['bannernotifications'][] = array(
+	'name' => 'BannerNotifications',
+	'descriptionmsg' => 'bannernotifications-desc',
+	'author' => 'Hyun Lim',
```

Comment:
Let's update this value since this extension has changed a bunch since Hyun last touched it.